### ISSUE TITLE
Translation of the interface to French and Flemish

### ISSUE THREAD

**Issue by npettiaux** [fossasia/open-event-server#6224]
I need, to respect the will of the sponsors as well as our attendees to have the interface completely translated to both French and Flemish as soon as possible. The event we deal with is 954136a on September 27.

We are ready to help with the translation itself.

Much thanks for your consideration

**Comment by mrsaicharan1** [comment]
I'll work on this.

**Comment by npettiaux** [comment]
Much thanks. As soon as you have .po file or other files that we (= me and my team) can translate to French first then to Dutch/Flemish, we'll start. Even one file at a time or part of such.

**Comment by npettiaux** [comment]
Indeed this is a front-end issue. The most important one is on the server side. Then on the web-app and the fdroid app as well.

**Comment by mrsaicharan1** [comment]
@npettiaux do you require any specific .po file?

**Comment by npettiaux** [comment]
Le vendredi 02 août 2019 à 02:22 -0700, Saicharan Reddy a écrit :
> @npettiaux do you require any specific .po file?

it would be even better & much simpler if you could synchonize such
files with the application offered online by crowdin.

Best regards
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
-- 
Nicolas Pettiaux, phd - nicolas@pettiaux.be - portable 0496 24 55 01
Educode - asbl - informer, former & réfléchir aux défis du numérique
https://educode.be & https://wiki.educode.be - ven 27/9 & lun 7/10 2019

**Comment by mariobehling** [comment]
A lot of translated strings have been added to the project through Weblate https://hosted.weblate.org/projects/open-event/frontend/. There are also French speaking events hosted nowadays on the system, e.g. https://eventyay.com/e/bcf0a8e9

Remaining is to translate the upcoming enhanced emails, so users will also receive email communication in French. Translation is an ongoing issue and we have established a workflow through Weblate for some time now. So, I am closing this issue. Thank you for pointing it out!
